uwblueprint/sistema
# add teacher management page

# Notion ticket link



[Unstyled Teacher Management](https://www.notion.so/uwblueprintexecs/Unstyled-Teacher-Management-355cd3aa265e46eaa9dfc81fcda9698b)



## Implementation description

- /manage page now contains teacher data
- can edit teacher role or delete teacher

![image](https://github.com/user-attachments/assets/5c61a220-3bb1-42db-bc58-2124daa3b5b8)



## Steps to test

1. ensure db works and is seeded (either docker or vercel)
2. go to /manage



## What should reviewers focus on?

- general good code practices

## Checklist
